cider/exec/nextgen/jitlib/llvmjit/LLVMJITModule.cpp

```diff
@@ -20,18 +20,52 @@
  */
 #include "exec/nextgen/jitlib/llvmjit/LLVMJITModule.h"
 
-#include <llvm/IR/Function.h>
+#include <llvm/Bitcode/BitcodeReader.h>
+#include <llvm/IR/Module.h>
+#include <llvm/Transforms/Utils/Cloning.h>
+
+#include <memory>
+#include <mutex>
 
 #include "exec/nextgen/jitlib/llvmjit/LLVMJITUtils.h"
 #include "util/Logger.h"
+#include "util/filesystem/cider_path.h"
 
 namespace cider::jitlib {
 
-LLVMJITModule::LLVMJITModule(const std::string& name)
+static llvm::MemoryBuffer* getExternalBuffer() {
```

Comment:
maybe better to rename external to runtime, eg. `getRuntimeBuffer` and `runtime_module` etc.

Reply:
get it